packages/mdc-checkbox/README.md

```diff
@@ -111,6 +111,35 @@ Note that `mdc-checkbox--disabled` is necessary on the root element of CSS-only
 <label for="basic-disabled-checkbox" id="basic-disabled-checkbox-label">This is my disabled checkbox</label>
 ```
 
+## Additional Information
+
+### Accessibility
+
+Material Design spec advises that touch targets should be at least 48 x 48 px.
+To meet this requirement, add the following to your checkbox:
```

Comment:
add the `mdc-checkbox--touch` class to your checkbox, as follows:

Reply:
Done